lib/duniverse.ml

```diff
@@ -133,6 +146,11 @@ module Deps = struct
     classify ~get_default_branch entries >>= fun classified -> Ok (from_classified classified)
 
   let count { opamverse; duniverse } = List.length opamverse + List.length duniverse
+
+  let resolve ~resolve_ref t =
+    let open Result.O in
+    Result.List.map ~f:(Source.resolve ~resolve_ref) t.duniverse >>= fun duniverse ->
+    Ok { t with duniverse }
```

Comment:
you can use map instead of bind here :)

Reply:
It is in fact intentional!

I find it more annoying and less diff friendly to have to update the infix operator when moving those lines around or further binding the result.

I agree in this case since there's only this one it would make sense but I prefer to adopt a consistent style and only use `Result.bind`.